hmy/api_backend.go

```diff
@@ -398,6 +400,9 @@ func (b *APIBackend) GetTotalStakingSnapshot() *big.Int {
 	stakes := big.NewInt(0)
 	for i := range candidates {
 		validator, _ := b.hmy.BlockChain().ReadValidatorInformation(candidates[i])
+		if !effective.IsEligibleForEPOSAuction(validator) {
```

Comment:
Why add this condition here? What is "GetTotalStakingSnapshot" used for?

Reply:
because otherwise you are counting the stake of banned or active=false validators for the median stake